src/components/Header/index.jsx

```diff
@@ -0,0 +1,38 @@
+import { NavLink } from 'react-router-dom';
+import DrJeannerotLogo from '../../assets/DrJeannerotLogo.png';
+import SHeader from './style';
+
+function Header() {
+  return (
+    <SHeader>
+      <div className="header">
```

Comment:
Why div with className="header" instead of a <header> ? And no need a className="header" with styled-components since it will target the header tag.